examples/advanced/transitive-transforms/tokens/color/font.json5

```diff
@@ -13,7 +13,7 @@
       },
       tertiary: {
         // transitive transforms allow you to modify a modified reference
-        value: "{color.font.secondary.value}",
+        value: "{color.font.secondary}",
```

Comment:
If we believe not having ".value" is the preferred method that we would like people to follow, this is great!

If not, it could be worthwhile to leave one of these with the .value appended so people know they can use it either way. Though, maybe that would be confusing and it is better to have a suggested method.